### PR TITLE
feat(vanilla-extract): Dialog, Divider, DoubleField, EmptyState, EmptyStateCard

### PR DESCRIPTION
**Pull request by sergivillar**
EmptyState component has been completely rewritten, IMO changes in the 2 broken screenshots are expected (As internal structure has been simplified, 1px has been moved around), if you are okey with this, I'll update that screenshot.

Dialog screenshot has been updated, now is more accurate. This is because previously the icon size was calculated used useScreenSize hook and `size` props from icon, now we have a wrapper around icon and styled with a media query (now size is 64px, previously 67, that's the difference)

### CODE REVIEW

File: src/empty-state.css.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+export const contentVariants = styleVariants({
+    default: {
+        marginTop: 0,
+        '@media': {
+            [mq.desktopOrBigger]: {
+                margin: 64,
+            },
+        },
+    },
+    largeImage: {
```
Comment (yceballost) [inline src/empty-state.css.ts:42]: I found differences in image behavior in mobile

production: [example 1](https://ibit.ly/lbCm) | [example 2 (1:1)](https://ibit.ly/5ibK)
this branch: [example 1](https://ibit.ly/JbUR) | [example 2 (1:1)](https://ibit.ly/Ezj_)

maybe missing `width 100%`?

I think the correct behavior should not show a deformed image, just should fit the photo to the longest side like the image component → https://ibit.ly/FHan

In desktop sizes seem to behave well

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/empty-state.css.ts (hunk 1 above)
Comment (sergivillar) [reply src/empty-state.css.ts:42]: @yceballost changes image behaviour test again in mobile and desktop (to be sure all still works as expected:) )

**Review by yceballost — APPROVED** [review]

File: src/double-field.css.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import {styleVariants} from '@vanilla-extract/css';
+import * as mq from './media-queries.css';
+import {sprinkles} from './sprinkles.css';
+import {DEFAULT_WIDTH} from './text-field-components';
```
[the remark this answers is not in the document]
Comment (sergivillar) [reply src/double-field.css.ts:4]: text-field-components is not migrated yet

File: src/empty-state.css.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        [mq.desktopOrBigger]: {
+            position: 'absolute',
+            height: '100%',
+        },
+    },
+});
+
+export const largeImage = style([
+    {
+        height: '100%',
```
Comment (pladaria) [inline src/empty-state.css.ts:63]: sprinkles

File: src/empty-state.tsx [hunk 4]
```diff
@@ -91,6 +48,14 @@ interface IconProps extends BaseProps {
     largeImageUrl?: undefined;
 }
 
+const getBorderStyle = (isInverse: boolean) => {
+    if (isInverse) {
+        return styles.inverseBorder;
+    }
+
+    return sprinkles({border: 'regular'});
```
Comment (pladaria) [inline src/empty-state.tsx:56]: I think this should be a style defined in the css

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Pull request merged**